draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -448,227 +348,153 @@ hop(AS(i), AS(j)) =  / Else, "Provider+" if the VAP-SPAS entry
+      </t>
       <t>
-        Looking at <xref target="fig2"/>, the UPDATE is received from a provider or a mutual-transit neighbor (i.e., AS(N) has that role in relation to the receiver).
-        The AS_PATH may have both an up-ramp (on the right starting at AS(1)) and a down-ramp (on the left starting at AS(N)).
-        The ramps are described as a sequence of ASes that consists of consecutive customer-to-provider hops.
-        The up-ramp starts at AS(1) and each AS hop, (AS(i), AS(i+1)), in it has the property that hop(AS(i), AS(i+1)) = "Provider+" for i = 1, 2,... , K-1.
-        If such a K does not exist, then K is set to 1.
-        The up-ramp ends (reaches its apex) at AS(K) because hop(AS(K), AS(K+1)) = "Not Provider+" or "No Attestation".
-        The down-ramp runs backward from AS(N) to AS(L).
-        Each AS hop, (AS(j), AS(j-1)), in it has the property that hop(AS(j), AS(j-1)) = "Provider+" for j = N, N-1,... , L+1.
-        If such an L does not exist, then L is set to N.
-        The down-ramp ends at AS(L) because hop(AS(L), AS(L-1)) = "Not Provider+" or "No Attestation".
-        Thus, the apex of the down-ramp is AS(L).
+        If the sum of lengths of up-rump and down-rump is less than N, it is Invalid.
       </t>
+
       <t>
-        If there is an up-ramp that runs across all ASes in the AS_PATH (i.e., K = N), then clearly the AS_PATH is Valid.
-        Similarly, if there is a down-ramp that runs across all ASes in the AS_PATH (i.e., L = 1), then also the AS_PATH is Valid.
-        However, if both ramps exist in an AS_PATH with K &lt; N and L > 1, then the AS_PATH is Valid if and only if L-K &lt;= 1.
-        Note that K could be greater than L (i.e., L-K has a negative value), which means that the up-ramp and down-ramp overlap, and that could happen when some adjacent ASes in the AS_PATH have mutual-transit relationship between them (i.e., include each other in their respective SPAS) (see <xref target="rec1"/>).
-        If L-K = 0, it means that the apexes of the up-ramp and down-ramp are at the same AS.
-        If L-K = 1, it means that the apexes are at adjacent ASes.
-        In summary, the AS_PATH is Valid if L-K is 0 or 1 or has a negative value.
+        With ASPA we can't measure exact length of the rumps, but we can measure it's minimum and maximum boundaries.
       </t>
       <t>
-        ------------
+        The maximum up-rump length can be defined as I, where I is the minimum index for which auth(A(I), A(I+1)) returns 'Not Provider'. If there is no such I, the maximum is equal to AS_PATH length, or N. We define this parameter as max_up_rump. The minimum up-rump length can be defined as I, where I is minimum index for which auth(A(I), A(I+1)) returns "No Attestation" or "Not Provider". If there is no such I, the AS_APATH consists only of "Provider+" pairs only, so the minimum up-rump length equals to AS_PATH length, or N. We define this parameter as min_up_rump.
```

Comment:
sed /rump/ramp/  typo ?

you named in ramp in the previous sections, why rename it here? Whats wrong with naming it ramp?

Reply:
typo, fixed